### PR TITLE
We had a hack night and created @CutePetsAtlanta

### PR DESCRIPTION
**Pull request by jcowhigjr**
http://beta.codeforatlanta.org/projects/4

### CODE REVIEW

File: where.geojson [hunk 1]
```diff
@@ -146,6 +146,18 @@
 				"type": "Point", 
 				"coordinates": [ -98.590635, 29.415142 ] 
 			}
-		}		
+		},
+				{
+			"type": "Feature",
+			"properties": 
+			{ 
+				"twitter" : "https://twitter.com/AtlantaCutePets"
```
Comment (birla22v) [inline where.geojson]: The twitter login has now been changed to "https://twitter.com/CutePetsAtlanta"

Comment (jcowhigjr) [inline where.geojson]: Thanks :+1:

**Comment by boonrs** [comment]
:v: Yay! May all the Atlanta pets be adopted. :camel: :shipit: :baby_chick: :hamster: :octopus: :rabbit: :racehorse:

**Comment by boonrs** [comment]
Could you merge the latest master into your branch and then we're good to merge!

**Comment by boonrs** [comment]
Close by pull request #26

**Pull request closed without merging**